api/controllers/ong.go

```diff
@@ -113,12 +113,23 @@ func (oc *OngController) FindByID(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Update Ong to database.
+// @Summary Update Ong By ID
+// @Description Retrieves Ong details based on the ong ID provided as a parameter.
```

Comment:
parece que ele vai apenas retornar e não fazer um update
```suggestion
// @Description Updates the details of an existing Ong based on the provided Ong ID.
```

Reply:
mas ta escrito duas vezes que vai update rs rs.

<img width="253" alt="Screenshot 2024-09-10 at 00 02 56" src="https://github.com/user-attachments/assets/a35a98cd-1777-4d01-80c8-21426a7ea79c">

Reply:
um é o comentário sobre a documentação, e o outro é a própria doc